cpp/arcticdb/storage/azure/azure_storage.cpp

```
@@ -59,6 +59,12 @@ void raise_azure_exception(const Azure::Core::RequestFailedException& e, const s
     auto status_code = e.StatusCode;
     std::string error_message;
 
+    // On some occasions, the SDK returns a non-200 status code even though there is no error
+    // this is not an error, so we return early
+    if (static_cast<int>(status_code) < 300) {
+        return;
```

Not even a debug statement, much less a warning? Just an entirely silent failure? Seems like saving up debugging fun for later